db/changes_view.go

```diff
@@ -25,46 +26,95 @@ type channelsViewRow struct {
 	}
 }
 
+func nextChannelViewEntry(results sgbucket.QueryResultIterator) (*LogEntry, bool) {
+
+	var viewRow channelsViewRow
+	found := results.Next(&viewRow)
+	if !found {
+		return nil, false
+	}
+
+	entry := &LogEntry{
+		Sequence:     uint64(viewRow.Key[1].(float64)),
```

Comment:
What about introducing two new methods on `channelsViewRow`?

```
func (v *channelsViewRow) getChanneName() {
   return v.Key[0]
}

func (v *channelsViewRow) getSequence() {
    return v.Key[1]
}
```

That seems like it would make it less error prone than `Key[1]`